src/xcode/ENA/ENA/Source/Scenes/Checkin/CheckinCoordinator.swift

```diff
@@ -18,11 +31,28 @@ final class CheckinCoordinator {
 			showQRCodeScanner: showQRCodeScanner,
 			showSettings: showSettings
 		)
-
-		return UINavigationController(rootViewController: checkInsTableViewController)
+		
+		// show the info screen only once
+		if !infoScreenShown {
+			return ENANavigationControllerWithFooter(rootViewController: infoScreen(hidesCloseButton: true, dismissAction: { [weak self] in
+				guard let self = self else { return }
+				self.infoScreenShown = true
```

Comment:
```suggestion
				guard let self = self else { return }
				self.viewController.pushViewController(self.overviewScreen, animated: true)	// Push Overview
				self.viewController.setViewControllers([self.overviewScreen], animated: false) // Set Overview as the only Controller on the navigation stack to avoid back gesture etc.
				self.infoScreenShown = true
```

Please push the view controller like we do in `DiaryCoordinator` and extract the overviewScreen into a `lazy var` just as we did there.

Reply:
We have to do this later. Currently there is no overview screen yet.

Reply:
ok, fixed this with commit 957318c